libs/designsystem/src/lib/components/list/list.component.spec.ts

```diff
@@ -142,16 +143,26 @@ describe('ListComponent', () => {
   describe('function: ngOnInit', () => {
     it('should enable load more, if there is a subscriber to the loadMore event emitter', () => {
       component.loadOnDemand.subscribe((_: LoadOnDemandEvent) => {});
-
       component.ngOnInit();
-
-      expect(component.isLoadOnDemandEnabled).toBeTruthy();
+      expect(component.isLoadOnDemandEnabled).toBeTrue();
```

Comment:
The line break is intentional to denote a arrange-act-assert pattern:
```suggestion

      expect(component.isLoadOnDemandEnabled).toBeTrue();
```

Reply:
Should be resolved by f1d733e.